cla_public/templates/moj_template_base.jinja

```diff
@@ -73,7 +73,8 @@
 
   </head>
 
-  <body class="govuk-template__body {% block body_classes %}{{ product_type }} {{ phase }}{% endblock %}">
+  <body class="govuk-template__body 
+  {% block body_classes %}{{ product_type }} {{ phase }} {{ 'cla-' + config["CLA_ENV"] if config["CLA_ENV"] != 'production' }} {% endblock %}">
```

Comment:
I would be somewhat tempted to remove the `if`, so we'd add the `cla-production` but just not necessarily do anything with it. Seems more predictable, I think?

Reply:
Done